Rocket.Chat/Models/Message/Message.swift

```diff
@@ -46,10 +46,6 @@ enum MessageType: String {
 }
 
 class Message: BaseModel {
-    // This time is related to the grouping feature
-    // we have in the list of messages
-    static var maximumTimeForSequence = 15.0 * 60.0 // 15 minutes
```

Comment:
This property is was used for unit testing, so now they're broken. Can you update the tests?

Reply:
Yes, sure! Fixed! :-)